src/core/output.cpp

```diff
@@ -279,6 +281,10 @@ namespace mamba
     /*****************
      * ConsoleStream *
      *****************/
+    ConsoleStream::ConsoleStream()
+    {
+        termcolor::colorize(*this);
```

Comment:
Maybe we should only do that if `stdout` `isatty` is true?

Reply:
> Maybe we should only do that if `stdout` `isatty` is true?

I implemented this because `ConsoleStream` destructor calls `Console::instance().print(str())` which is *always* printing to `stdout`, but the intermediate stream will not be detected as a tty (because it's not), preventing `termcolor` to add ansi colors

> There is also the env var CLICOLOR_FORCE=1 and if it's set we should always force colors :)

do you want to add some `CLICOLOR_FORCE` flag in `libmamba`?